servicex/db_utils.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy import DateTime
+from sqlalchemy.sql import expression
+from sqlalchemy.ext.compiler import compiles
+
+
+# Tell database to generate a UTC timestamp
```

Comment:
Is adding all of this really necessary when you could just use the python datetime library to populate the field?

Reply:
It's not strictly necessary because we don't particularly care about having exact times, but my understanding is that having the database calculate the times is a best practice (this [SO post](https://stackoverflow.com/a/33532154) is what convinced me).

Reply:
https://deviq.com/yagni/